lib/jmap/core/method/request/event_action_method.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/method/method.dart';
+
+abstract class EventActionMethod extends MethodRequiringAccountId {
```

Comment:
- Rename to `CalendarEventReplyMethod `
- Extend for `Accept/Reject/Maybe`

Reply:
Done